laravel-api/infrastructure/Http/Kernel.php

```diff
@@ -18,6 +18,8 @@ class Kernel extends HttpKernel
         \Infrastructure\Http\Middleware\EncryptCookies::class,
         \Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse::class,
         \Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull::class,
+        \Illuminate\Session\Middleware\StartSession::class,
+        \Illuminate\View\Middleware\ShareErrorsFromSession::class,
```

Comment:
Check how password reset form behaves when bad data entered and validation doesn't pass